lego/apps/users/tests/test_models.py

```diff
@@ -426,5 +430,22 @@ def test_get_consents_with_prior_consents(self):
         )
         initial_consents = self.test_user.photo_consents.all()
         self.assertEqual(len(initial_consents), 2)
-        user_consents = PhotoConsent.get_consents(self, self.test_user)
+        user_consents = PhotoConsent.get_consents(self.test_user)
         self.assertEqual(len(user_consents), 2)
+
+    def test_get_consents_for_past(self):
+        """
+        It should not be possible to generate new consents for the past.
+        """
+        user_consents = PhotoConsent.get_consents(self.test_user)
+        self.assertEqual(len(user_consents), 2)
+
+        today = timezone.now()
```

Comment:
Kind of unnecessary variable, especially considering it's only used to calculate a different value.